loom/engine/loom2d/l2dDisplayObject.h

```diff
@@ -151,6 +151,7 @@ class DisplayObject : public EventDispatcher
         valid          = false;
         type           = NULL;
         imageOrDerived = false;
+        transformDirty = false;
```

Comment:
Yeah I wasn't sure, `true` seems to make more sense, but default boolean is usually `false`. I assume in release builds it was just random?